package.json

```diff
@@ -57,6 +57,28 @@
       "description": "New tabs are opened at the top",
       "type": "bool",
       "value": false
+    },
+    {
+      "name": "largetabs",
+      "hidden": true,
+      "title": "Large Tabs",
+      "description": "Tabs are displayed large with thumbnail",
```

Comment:
Hmm, okay, then how about "Display thumbnails in tabs", and change "Auto" to "When there's room"…